verification.py

```diff
@@ -2220,6 +2220,47 @@ def verify_breakpoint_distribution(
         pyplot.savefig(path)
         pyplot.close('all')
 
+    def verify_recombination(
+            self, basedir, name, sample_size, Ne, r, L, model, growth_rate=0):
+        """
+        Verifies that the number of recombination equals the number of mutation.
+        """
+        if not os.path.exists(basedir):
+            os.mkdir(basedir)
+        empirical_theta = []
+        empirical_rho = []
+        for i in range(1,200):
+                ts = msprime.simulate(
```

Comment:
If we use ``msprime.simulator_factory()`` to get a simulator, we should be able to use the num_breakpoints from there.